pandas/tests/io/json/test_json_table_schema.py

```diff
@@ -845,3 +845,13 @@ def test_read_json_orient_table_old_schema_version(self):
         expected = DataFrame({"a": [1, 2.0, "s"]})
         result = pd.read_json(StringIO(df_json), orient="table")
         tm.assert_frame_equal(expected, result)
+
+    def test_read_json_table_orient_period_depr_freq(self, recwarn):
+        # GH#9586
+        df = DataFrame(
+            {"ints": [1, 2]},
+            index=pd.PeriodIndex(["2011-01", "2011-08"], freq="2M"),
```

Comment:
could we check it works for both `'2M'` and `'M'`?

Reply:
yeah, sure, I added parameterization to the test, for `'M'` and `'2M'`.